query/apimanagement.sp

```diff
@@ -1,15 +1,15 @@
 query "apimanagement_service_with_virtual_network" {
   sql = <<-EOQ
     select
-      type || ' ' || name as resource,
+      type || ' ' || address as resource,
```

Comment:
```suggestion
      address as resource,
```
@rajlearner17, we don't need to append the `type` column to the `address`. We will use only the `address` as `resource`.